client/src/app/site/motions/modules/motion-workflow/components/workflow-list/workflow-list.component.ts

```diff
@@ -80,8 +80,7 @@ export class WorkflowListComponent extends BaseListViewComponent<ViewMotionWorkf
                 {
                     idField: 'motion_workflow_ids'
                 }
-            ],
-            fieldset: []
+            ]
```

Comment:
Please undo these changes - the `fieldset: []` clears required fields for the ViewMeeting. If it is not gives, the DEFAULT_FIELDSET will be used, which will currently fetch the `description` of the meeting, but it is not needed here.